src/cli/onefuzz/api.py

```diff
@@ -1393,8 +1393,9 @@ def _expand_scaleset_machine(
     def create(
         self,
         pool_name: primitives.PoolName,
-        size: int,
+        max_size: int,
         *,
+        intialized_size: Optional[int] = 1,
```

Comment:
I think `initial_size` makes a bit more sense because the scaleset has not been created yet, so a past tense parameter is a little confusing.  We want to communicate that this is the size we will initialize the scale set to.

Reply:
Oh I much prefer that. Good call.